string boxName
string baseTopic
---
bool success